api/nodes/views.py

```diff
@@ -245,11 +246,7 @@ def get_default_odm_query(self):
 
     def filter_non_retracted_nodes(self, query):
```

Comment:
This method is superfluous now that we're no longer filtering out retractions here.